packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -123,17 +123,11 @@ namespace Tpetra {
     /// \brief Status of the graph's or matrix's storage, when not in
     ///   a fill-complete state.
     ///
-    /// When a CrsGraph or CrsMatrix is <i>not</i> fill complete, its
-    /// data live in one of three storage formats:
+    /// When a CrsGraph or CrsMatrix is <i>not</i> fill complete and
+    /// is allocated, then its data live in one of three storage
```

Comment:
There are only 2 formats now

Reply:
@brian-kelley Nice catch! : - D  I'll fix this first.